### PR TITLE
Issue 1497 - Change the CLI to support deployment policy as an alias for business policy

### PR DESCRIPTION
**Pull request by MaxMcAdam**

### CODE REVIEW

File: cli/hzn.go [hunk 1]
```diff
@@ -186,14 +186,14 @@ Environment Variables:
...
+	exAgbotListPolicyCmd := exAgbotCmd.Command("listdeploymentpol", msgPrinter.Sprintf("Display the deployment policies that this agbot is serving."))
+	exAgbotPol := exAgbotListPolicyCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to list serving deployment policies for.")).Required().String()
+	exAgbotAddPolCmd := exAgbotCmd.Command("adddeploymentpol", msgPrinter.Sprintf("Add this deployment policy to the list of policies this agbot is serving. Currently only support adding all the deployment policies from an organization."))
+	exAgbotAPolAg := exAgbotAddPolCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to add the deployment policy to.")).Required().String()
+	exAgbotAPPolOrg := exAgbotAddPolCmd.Arg("policyorg", msgPrinter.Sprintf("The organization of the deployment policy to add.")).Required().String()
+	exAgbotDelPolCmd := exAgbotCmd.Command("removedeploymentpol", msgPrinter.Sprintf("Remove this deployment policy from the list of policies this agbot is serving. Currently only support removing all the deployment policies from an organization."))
```
Comment (linggao) [inline cli/hzn.go]: Could you also add alias for listdeploymentpol, adddeploymentpol and removedeploymentpol to support backward compatibility?

File: cli/hzn.go [hunk 2]
```diff
@@ -610,13 +616,13 @@ Environment Variables:
 			credToUse = cliutils.GetExchangeAuth(*exUserPw, *exServiceAddPolicyIdTok)
 		case "service removepolicy":
 			credToUse = cliutils.GetExchangeAuth(*exUserPw, *exServiceRemovePolicyIdTok)
-		case "business listpolicy":
```
Comment (linggao) [inline cli/hzn.go]: If user run `hzn exchange business listpolicy` does it go the the `deployment` case?

File: cli/hzn.go (hunk 1 above)
Comment (MaxMcAdam) [reply cli/hzn.go]: Added

File: cli/hzn.go (hunk 2 above)
Comment (MaxMcAdam) [reply cli/hzn.go]: Yes, it does

**Pull request merged**